tools/crm_mon_print.c

```diff
@@ -1085,7 +1096,7 @@ print_status(pcmk__output_t *out, mon_output_format_t output_format,
     }
 
     /* If there were any failed actions, print them */
-    if (xml_has_children(data_set->failed)) {
+    if (is_set(show, mon_show_fencing) && xml_has_children(data_set->failed)) {
```

Comment:
This is unrelated -- these are failed resource actions, not failed fence actions. I suppose we need a new section option for it.

Reply:
Added.  It is not shown by default except for XML output, but that would be easy to change.

Reply:
All the formats show the section by default (if any failed actions exist)